dts/bindings/display/galaxycore,gc9x01x.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2023 Mr Beam Lasers GmbH.
+# Copyright (c) 2023 Amrith Venkat Kesavamoorthi <amrith@mr-beam.org>
+# Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  GC9X01X display driver.
+
+  This driver implements support for various GC9xxx graphics
+  controllers and different display sizes. It has been validated
+  for following controllers:
+   - GC91A01A: (Waveshare 240x240, 1.28inch round lcd display 240x240)
```

Comment:
```suggestion
   - GC9A01A: (Waveshare 240x240, 1.28inch round lcd display 240x240)
```

Reply:
OMG! It looks like I am meanwhile code blind and just react on comments.
Fixed in [c045f5c](https://github.com/zephyrproject-rtos/zephyr/commit/c045f5ce0a5f4aa4fb68a0bd7804e7944cc6564b)